runtime/dom-context.js

```diff
@@ -52,6 +59,9 @@ class DomContext {
     this._innerContextBySlotName = {};
 
   }
+  static createTemplateElement(template) {
+    return Object.assign(document.createElement('template'), { innerHTML: template});
```

Comment:
Non-symmetric spacing around `innerHTML` (whatever, I noticed it, lol).

Reply:
done